test/validate/database_test.py

```diff
@@ -1,12 +0,0 @@
-"""Validate database integrity checks."""
-from dagster import build_init_resource_context
-
-from pudl.io_managers import pudl_sqlite_io_manager
-
-
-def test_foreign_key_constraints():
-    """Check foreign key constraints of current database."""
-    # TODO (bendnorman): Replace this with a fixture like the live_dbs fixtures
```

Comment:
I'm guess this happened and this function just didn't get deleted?

Reply:
Not sure. @bendnorman mentioned this test is obsolete so I just deleted it.